### PR TITLE
Translate to Danish

### PR DESCRIPTION
**Pull request by coffandro**
I have translated index.md to Danish

### Type
- [x] Rewrite already written documentation.
- [ ] Add/remove new pages.

### Questions
- [ ] Did you preview on your local machine beforehand?
- [x] Did you follow our [commit handbook](https://github.com/Atlas-OS/docs/blob/master/.github/CONTRIBUTING.md#commit-message)?
- [x] Did you read the [Atlas Contribution Guidelines](https://docs.atlasos.net/contributions/)?
- [x] Did you commit to the `dev` branch and not `master`?

### Describe your pull request
I have translated index.md to Danish, I will continue translating some other pages but currently this is it

### CODE REVIEW

File: docs/da/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Velkommen til Atlas' Dokumentation
+
+<div class="grid cards" markdown>
+
+-   :material-power:{ .lg .middle } __Installer AtlasOS__
+
+    ---
+
+    start op med at få AtlasOS sat op på dit system på blot et par minutter.
```
Comment (whallin) [inline docs/da/index.md:14]: Is "start" not supposed to be with an upper-case S?

File: docs/da/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    [:octicons-arrow-right-24: Install AtlasOS](getting-started/installation.md)
```
Comment (whallin) [inline docs/da/index.md:16]: The link name here, "Install AtlasOS", can also be translated if there is a translation available in Danish. This applies to the 3 other links further down in the file, too. :)

**Review by whallin — CHANGES_REQUESTED** [review]
I think it looks good! I mentioned a few things that you might want to look into.

Also, could you please follow the steps in the link below to add Danish as a language to the documentation the right way? Your translations will not load properly on build without the configuration changes.

https://i18n.atlasos-docs-enq.pages.dev/localization/#adding-a-new-language

File: docs/da/index.md (hunk 1 above)
Comment (coffandro) [reply docs/da/index.md:14]: Yeah I didn't notice that, I'll fix it on the next version

File: docs/da/index.md (hunk 2 above)
Comment (coffandro) [reply docs/da/index.md:16]: Oh, didn't see that lol

**Comment by Ast3risk-ops** [comment]
Currently i18n has been depreceated in favour of Google Translate.

**Comment by coffandro** [comment]
yup, had abandoned it when you told me in the Discord server

**Pull request closed without merging**